roles/keepalived/templates/haproxy_check.sh.j2

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+CURRRENT_ROLE=$(curl http://{{ hostvars[host]['inventory_hostname'] }}:8008/master --silent | jq ' .role' | sed 's/"//g')
```

Comment:
Why do you think that VIP should be run on a server with the role of master (leader)?

For the Type A scheme, it does not matter which of the servers is responsible for balancing

In the README you can find the following description for keepalived:

> In our configuration keepalived checks the status of the HAProxy service and in case of a failure delegates the VIP to another server in the cluster.

This is important to ensure the high availability of HAProxy

/bin/kill -0 `cat /run/haproxy/haproxy.pid`

I suggest not to make changes to this logic.

Reply:
I think it makes sense for dcs_type == 'patroni_raft' 

Perhaps we could make a condition and add this check (leader role) only for patroni_raft

Reply:
> sed 's/"//g'

try `jq -r` instead of using sed